experiments/list_unsupported.sh

```diff
@@ -34,8 +34,35 @@ done
 
 echo "$1: Unsupported features\n" > "$file_name".txt
 
+if ! command -v gnat > /dev/null; then
+   echo >&2 "Gnat not on PATH!"
+   exit 4
+fi
+
+ADA_HOME=`which gnat`
+ADA_HOME="$(dirname "$ADA_HOME")/.."
+DEF_ADA_INCLUDE_PATH="${ADA_HOME}/lib/gcc/x86_64-pc-linux-gnu"
```

Comment:
No chance of a `uname -m` instead of the hard coded `x86_64` is there?

Reply:
Good point!

Reply:
Agreed. Modified.

Reply:
Ta.